.github/workflows/CI.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           environment-name: rmg_env
           channels: defaults,rmg,conda-forge
           channel-priority: flexible
-          log-level: debug
+          log-level: warning
```

Comment:
Why do we reset log level to `warning` here?

Reply:
Can you squash this with the commit that set it to debug?

Reply:
Debug writes more to stdout than is useful for any future test running (see [this run with debug](https://github.com/ReactionMechanismGenerator/RMG-Py/actions/runs/4097452798/jobs/7065836940#step:3:82)) -- I only turned it on to deal with the issue of channel-priority. The default log level is warning